src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  background-color: $color-white;
+  border: 1px solid $color-light;
+  padding: 32px 16px 16px ;
+
+  &__image {
+    height: 134px;
+    margin: 0 3px 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0 0 4px;
+    color: $color-main;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    height: 14px;
+    color: $color-secondary;
+    margin: 0 0 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__comments {
+    font-weight: 400;
+    font-size: 10px;
+    color: $color-main;
+    margin: 0;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__text {
+    margin: 0;
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-secondary;
+  }
+
+  &__value {
+    margin: 0;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $color-main;
+    text-align: right;
+  }
+
+  &__button {
+    display: flex;
+    font-weight: 700;
+    font-size: 14px;
+    height: 40px;
+    text-transform: uppercase;
+    text-decoration: none;
+    justify-content: center;
+    align-items: center;
+    color: $color-white;
+    background-color: $color-blue;
+    border: 1px solid $color-blue;
+    border-radius: 5px;
+  }
+
+  transition: all 300ms;
```

Comment:
Don't use all for transition property. Specify the properties instead

Reply:
Ok